### ISSUE TITLE
Trying Charge and error on serve/build

### ISSUE THREAD

**Issue by mrpotatoes** [brandonweiss/charge#808]
My process

```
git clone https://github.com/brandonweiss/charge.git
cd charge
yarn install
node cli.js [build|serve] docs/ tmp/
```

Receive this error for either `build` or `serve`:
```
Error: Unknown language: `text` is not registered
```

I want to mention that I'm really interested in Charge and hope it becomes bigger. I may be forking to see what I can get into it (generalize for myself and others Charge included) so I'd like to be able to play around with it more.

**Comment by mrpotatoes** [comment]
Found the issue. There is no in the triple back tick the `text` language isn't supported. I'll make a quick pull request to fix that and add more informative error output.

**Comment by mrpotatoes** [comment]
https://github.com/brandonweiss/charge/pull/809

**Comment by mrpotatoes** [comment]
I'm actually interested why the `importComponent` function is even throwing this error when there is language syntax that isn't registered. How does it know? How are they registered?

**Comment by brandonweiss** [comment]
Hey! Oh no 🙈 That’s terrible timing—I made that change like just a day ago 🤦 Thank you for fixing it!

I would have expected Netlify to let me know the docs deployment failed—I guess that’s something I have to manually set up 🤔 Done!

As to how it’s throwing the error, it’s a combination of it not being normal Markdown (it’s MDX) and I’m doing the transpilation/compilation with require hooks. So when the file is imported/required it’s actually parsing the markdown as MDX, and it throws an error if something is wrong.

**Comment by brandonweiss** [comment]
Fixed in #809.
